index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
```

Comment:
> Good use of semantic HTML tags, try to add comments at the beginning of new sections to indicate the start of new section, try and

- [ ] I would suggest you make it polite by adding 'I would suggest you try to...' so that it would not be seen as a command but a polite request.